addons/nuxeo-platform-document-routing/nuxeo-routing-core/src/test/java/org/nuxeo/ecm/platform/routing/test/WorkflowEscalationTest.java

```diff
@@ -309,6 +314,69 @@ public void testEscalationMultipleExecution() throws Exception {
+    public void testGetOpenTasksOperation() throws InterruptedException {
+        // define a start node with an escalation rule
+        DocumentModel node1 = createNode(routeDoc, "node1", session);
+        node1.setPropertyValue(GraphNode.PROP_START, Boolean.TRUE);
+        setTransitions(node1, transition("trans1", "node2"));
+        node1.setPropertyValue(GraphNode.PROP_HAS_TASK, Boolean.TRUE);
+        node1.setPropertyValue(GraphNode.PROP_TASK_DUE_DATE_EXPR, "CurrentDate.days(-1)");
+        setEscalationRules(node1, escalationRule("rule1", "WorkflowFn.timeSinceDueDateIsOver() >=3600000",
+                "test_completeOpenTasks", false));
+        session.saveDocument(node1);
+
+        // define an end node
+        DocumentModel node2 = createNode(routeDoc, "node2", session);
+        node2.setPropertyValue(GraphNode.PROP_MERGE, "all");
+        node2.setPropertyValue(GraphNode.PROP_STOP, Boolean.TRUE);
+        session.saveDocument(node2);
+
+        // start the workflow
+        DocumentRoute routeInstance = instantiateAndRun(session);
+        String routeInstanceId = routeInstance.getDocument().getId();
+
+        transactionalFeature.nextTransaction();
+
+        // check suspended nodes with escalation rules
+        List<String> nodes = escalationService.queryForSuspendedNodesWithEscalation(session);
+        assertEquals(1, nodes.size());
+        DocumentRef nodeDocRef = new IdRef(nodes.get(0));
+        DocumentModel nodeDoc = session.getDocument(nodeDocRef);
+        GraphNode node = nodeDoc.getAdapter(GraphNode.class);
+        assertEquals("node1", node.getId());
+
+        // compute and schedule escalation rule
+        List<GraphNode.EscalationRule> rules = escalationService.computeEscalationRulesToExecute(node);
+        assertEquals(1, rules.size());
+        escalationService.scheduleExecution(rules.get(0), session);
+
+        transactionalFeature.nextTransaction();
+
+        // fetch node doc to check that the escalation rule is marked as executed
+        nodeDoc = session.getDocument(nodeDocRef);
+        node = nodeDoc.getAdapter(GraphNode.class);
+        assertTrue(node.getEscalationRules().get(0).isExecuted());
+
+        // check that the rule was executed
+        assertEquals(0, taskService.getAllTaskInstances(routeInstanceId, nodeDoc.getId(), session).size());
+
+        // check that no nodes with escalation rules are found
+        nodes = escalationService.queryForSuspendedNodesWithEscalation(session);
+        assertEquals(0, nodes.size());
+
+        // cancel the route
+        routingService.cancel(routeInstance, session);
+        routeInstance = session.getDocument(new IdRef(routeInstanceId)).getAdapter(DocumentRoute.class);
+        assertTrue(routeInstance.isCanceled());
+    }
+
     protected void setEscalationRules(DocumentModel node, Map<String, Serializable>... rules) {
         node.setPropertyValue(GraphNode.PROP_ESCALATION_RULES, (Serializable) Arrays.asList(rules));
```

Comment:
i think something happened here after replacing the `Array.asList` by `List.of` in your previous commit (clean up one)